swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2020-12-31-preview/redhatopenshift.json

```diff
@@ -735,7 +735,11 @@
         "Standard_F4s_v2",
         "Standard_F8s_v2"
       ],
-      "type": "string"
+      "type": "string",
+      "x-ms-enum": {
```

Comment:
surely the older api stays the same?

Reply:
According to documentation, the old part stays the same in case the consumer does not understand the new api